coalib/testing/LocalBearTestHelper.py

```diff
@@ -52,6 +54,18 @@ class LocalBearTestHelper(unittest.TestCase):
     If you miss some methods, get in contact with us, we'll be happy to help!
     """
 
+    def assertObjectsEqual(self, observed_result, expected_result):
+        attributes = Result.__compare_fields__
+        for observed, expected in zip(observed_result, expected_result):
+            if isinstance(observed, Comparable) and isinstance(
+                    expected, Comparable):
+                for attribute in attributes:
```

Comment:
see above, get the attributes dynamically :)
`type(observed).__compare_fields__`